plugins/linux-v4l2/v4l2-output.c

```diff
@@ -146,16 +197,24 @@ static bool try_connect(void *data, const char *device)
 	parm.parm.output.timeperframe.numerator = ovi.fps_den;
 	parm.parm.output.timeperframe.denominator = ovi.fps_num;
 
-	if (ioctl(vcam->device, VIDIOC_S_PARM, &parm) < 0)
+	if (ioctl(vcam->device, VIDIOC_S_PARM, &parm) == -1) {
+		blog(LOG_WARNING,
```

Comment:
Also - see comment at line 148 about the debug output that is available with v4l2loopback.